drivers/hwinfo/Kconfig

```diff
@@ -32,6 +32,7 @@ config HWINFO_NRF
 	bool "NRF device ID"
 	default y
 	depends on SOC_FAMILY_NRF
+	depends on SOC_SECURE_NRF
```

Comment:
Is the name a bit of a misnomer in the context of nRF devices that do not support trustzone?

Reply:
I am told it's the name of the header :)